src/EnergyPlus/PluginManager.cc

```diff
@@ -514,18 +512,17 @@ PluginManager::PluginManager(EnergyPlusData &state) : eplusRunningViaPythonAPI(s
                 // defaulted to YES
             }
             if (epInDirFlagUC == "YES") {
-                std::string epin_path;
+                std::string epin_path; // NOLINT(misc-const-correctness)
```

Comment:
It really needed you to tell it that epin_path was not const?  Weird.

Reply:
clang-tidy can't see past the Optional here: https://github.com/NREL/EnergyPlus/blob/b7a30cf169f81fec733cee276d77bb4b8152dc64/third_party/ObjexxFCL/src/ObjexxFCL/environment.cc#L54.